src/django/pfb_analysis/management/commands/import_results_shapefiles.py

```diff
@@ -74,23 +75,13 @@ def import_shapefile(job, shpfile_name, model, layer_mapping):
                                             import_file,
                                             layer_mapping)
             import_layer_map.save()
-        except LayerMapError as e:
-            # If we failed with an error related to JOB_ID not found, it's because this is an
-            # old job that doesn't contain the column in the shapefile. So we try again with
```

Comment:
Are imports of jobs before the export column was added no longer supported now then? If it fails due to it missing, will that be clear to the user?

Reply:
Oh, it's not going to be used at all, okay.